homeassistant/components/uptimerobot/__init__.py

```diff
@@ -57,6 +57,8 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 class UptimeRobotDataUpdateCoordinator(DataUpdateCoordinator):
     """Data update coordinator for UptimeRobot."""
 
+    data: list[UptimeRobotMonitor]
```

Comment:
The `DataUpdateCoordinator` is a generic and allows typing the `data` attribute by defining the class like so:

```py
class UptimeRobotDataUpdateCoordinator(DataUpdateCoordinator[list[UptimeRobotMonitor]]):
```